source/crud/crud.rst

```diff
@@ -227,6 +227,13 @@ Read
      */
     useCursor: Optional<Boolean>;
 
+    /**
+     * Enables users to specify an arbitrary string to help trace the operation through
+     * the database profiler, currentOp and logs. The default is to not send a value.
+     *
+     * @see http://docs.mongodb.com/manual/reference/command/aggregate/
+     */
+    comment : Optional<String>;
```

Comment:
Space between "comment" and colon can be removed for consistency.

I think we also need to add a changelog entry at the bottom of the file, but that can likely be done immediately before merging.